test/test_numbers.py

```diff
@@ -0,0 +1,52 @@
+import os

```

Comment:
I'm pretty sure we don't use `os`

I also don't think we use lines 5-8, where we get at `reptest` files.